pyemd.c

```diff
@@ -120,8 +130,20 @@ static PyObject *compute_emd(PyObject *self, PyObject *args, PyObject *keywds)
   signature2.Features = f2;
   signature2.Weights = w2;
 
-  distance = emd(&signature1, &signature2, double_dist, 0, 0);
+  distance = emd(&signature1, &signature2, double_dist, flow, &flow_size);
 
+  if (return_flows){
+	  PyObject *returnList = PyList_New((Py_ssize_t) flow_size);
```

Comment:
Mixed tabs and spaces here, while the rest of the code uses 2-space indentation. Not a huge deal, but if you have time to fix would be nice.